utils/token.ts

```diff
@@ -0,0 +1,11 @@
+import { ethers } from 'ethers';
+
+export const ethersParsable = (tokenAmount: string, decimals: number) => {
```

Comment:
nit: since this returns a bool maybe this name change is clearer

```suggestion
export const isAmountParsable = (tokenAmount: string, decimals: number) => {
```